src/Form/AnalyzeSettingsForm.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\analyze\Form;
+
+use Drupal\analyze\HelperInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Element;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure Analyze settings for this site.
+ */
+final class AnalyzeSettingsForm extends ConfigFormBase {
+
+  /**
+   * Constructs a \Drupal\system\ConfigFormBase object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The factory for configuration objects.
+   * @param \Drupal\Core\Config\TypedConfigManagerInterface|null $typedConfigManager
+   *   The typed config manager.
+   * @param \Drupal\analyze\HelperInterface $helper
+   *   The Analyze Helper service.
+   */
+  public function __construct(
+    ConfigFactoryInterface $config_factory,
+    protected readonly HelperInterface $helper,
+    protected $typedConfigManager = NULL,
```

Comment:
After installing this on D11/php 8.3, I get the following error messages and also when flushing the cache.

`PHP Fatal error:  Type of Drupal\analyze\Form\AnalyzeSettingsForm::$typedConfigManager must be Drupal\Core\Config\TypedConfigManagerInterface (as in class Drupal\Core\Form\ConfigFormBase) in /var/www/html/web/modules/custom/analyze/src/Form/AnalyzeSettingsForm.php on line 17`

The typing has to be set there, so it should be something like
`protected TypedConfigManagerInterface $typedConfigManager`